scripts/stow.sh

```
@@ -22,21 +22,72 @@ else
 STOW_DIR="$DOTFILES_DIR/stow"
-find "$STOW_DIR" -type f ! -name .DS_Store | while read -r file; do
+STOW_IGNORE=()
+
+# Exclude one file from this stow run. --ignore takes a Perl regex matched
+# against the basename, so the dots have to be escaped.
+skip_file() {
+  STOW_IGNORE+=(--ignore="$(basename "$1" | sed 's/\./\\./g')")
+}
+
+while read -r file; do
   rel_path="${file#$STOW_DIR/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)